Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.cxx

```diff
@@ -443,6 +446,49 @@ int vtkMRMLVolumeArchetypeStorageNode::ReadDataInternal(vtkMRMLNode *refNode)
 
   vtkNew<vtkImageData> iciOutputCopy;
   iciOutputCopy->ShallowCopy(ici->GetOutput());
+  vtkInformation* imgInfo = iciOutputCopy->GetPointData() ?
```

Comment:
Yeah sorry, this is still experimental/testing for now, I'm just pushing things to keep them online (that's why I have 3 ugly commits): I wanted to know where we want to put it first: in the storagenode or in the reader?